docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -70,28 +70,42 @@ Finally, you can query the ``clicks`` table in the ``web`` schema::
 If you used a different name for your catalog properties file, use
 that catalog name instead of ``sqlserver`` in the above examples.
 
+.. _sqlserver-performance:
+
+Performance
+-----------
+
+The connector includes a number of performance improvements, detailed in the
+following sections.
+
+* Support for :ref:`JDBC connection pooling <base-jdbc-connection-pooling>`
```

Comment:
Does SQL Server support pooling?

Reply:
I think yes.. but it would probably have to be confirmed

Reply:
So i did not find any evidence of pooling there (hence my comment).
It seems to me totally made up, but maybe you have some reference for this.

Reply:
<img width="442" alt="Screen Shot 2021-04-09 at 9 11 03 AM" src="https://user-images.githubusercontent.com/1664457/114209532-8fbeb300-9913-11eb-8bd8-d318fe550210.png">

Reply:
Only oracle in Trino.

Reply:
<img width="462" alt="Screen Shot 2021-04-09 at 9 42 00 AM" src="https://user-images.githubusercontent.com/1664457/114213469-db735b80-9917-11eb-880d-acfe5770538a.png">

Reply:
Yes, I am aware & confirm that Oracle connector supports pooling.

Reply:
So I will move this out of the PR for now.. I guess it should probably move from the Oracle connector to the base jdbc and then get farmed out to all the other jdbc connectors.